src/buildTree.js

```diff
@@ -13,6 +13,9 @@ const optionsFromSettings = function(settings: Settings) {
     return new Options({
         style: (settings.displayMode ? Style.DISPLAY : Style.TEXT),
         maxSize: settings.maxSize,
+        minRuleThickness: isNaN(settings.minRuleThickness)
```

Comment:
I also did not think it would be necessary. But I did get a NaN before I wrote this line and I'm not sure why. I'll take it out if I can fix things elsewhere so that this is not needed.

Reply:
Done.